docs/book/provider_implementations/register_schemes.md

```diff
@@ -51,6 +54,18 @@ func main() {
         metricsAddr := flag.String("metrics-addr", ":8080", "The address the metric endpoint binds to.")
         flag.Parse()
 
+        klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+        klog.InitFlags(klogFlags)
```

Comment:
Is it ok to call `klog.InitFlags` twice like this?

I'm also wondering if we need the glog workaround still, since the latest dependencies shouldn't be bringing in glog from the k/k ecosystem anymore. @vincepri thoughts?

Reply:
Removed duplicate `InitFlags` call and glog sync - these are not required